### ISSUE TITLE
SHIFT+CRSR doesn't select in MS-DOS EDIT

### ISSUE THREAD

**Issue by maxpat78** [joncampbell123/dosbox-x#4588]
### Describe the bug

MS-DOS editor is supposed to select and highlight lines by pressing SHIFT and cursor up/down keys.

However, when AUTOEXEC.BAT executes `KEYB IT,,C:\DOS\KEYBOARD.SYS` (a default command to set the right keyboard layout in an italian environment) or simply `KEYB IT`, such key combination stops working.

Also, resetting the builtin US keyboard with `KEYB US` does not fix.

### Steps to reproduce the behaviour

Load KEYB.COM in MS-DOS 6.22

### Expected behavior

Standard MS-DOS KEYB.COM should work fine.

### What operating system(s) this bug have occurred on?

Windows 11 x64 22H2

### What version(s) of DOSBox-X have this bug?

2023.10.06

### Used configuration

```ini
Both SDL1 & SDL2
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
It seems to work as expected?
Tried the latest commit, by launching dosbox-x with option `-defaultconf` and booted an image with MS-DOS6.22 installed.
Maybe I'm not following your steps.

https://github.com/joncampbell123/dosbox-x/assets/68574602/a64cf452-c478-4861-be1a-13e72313c7bf

**Comment by maxpat78** [comment]
@maron2000 This is from 2023.10.06 Release downloaded from site:

https://github.com/joncampbell123/dosbox-x/assets/2823136/13a9260e-0c20-42b9-bf48-6c69cf33f77c

The same with latest x64 SDL1-2 commits built this morning!

**Comment by maron2000** [comment]
I believe you booted from a setup disk, so I tried it as well.
Boot Setup disk1 -> abort setup -> `edit autoexec.bat`
I also tried running `keyb it,,keyboard.sys` before edit, but works fine as well.
Very strange.

![edit2](https://github.com/joncampbell123/dosbox-x/assets/68574602/7e3f9079-ff53-4db7-88e1-7b92eceb55ac)

**Comment by maxpat78** [comment]
@maron2000 The same happens to me booting from an HD image.

I've also tried `-defaultconf`, and to run DosBox-X 2023.10.06 from another PC (a laptop): but selection/highlight does not work (it's not matter of mere transparency lack).

Changing video output seems without effect on the problem, too.

**Comment by maxpat78** [comment]
And the same issue occurs with MS-DOS 5, also.

**Comment by maxpat78** [comment]
@maron2000 Are you using USA MS-DOS floppies?

**Comment by maron2000** [comment]
I tried Italian MSDOS 6.22 and even set the host keyboard to Italian, but I really can't reproduce your issue.

**Comment by maxpat78** [comment]
I can't understand, I've retried on a 3rd PC (Windows 10 laptop), but it's the same!

Perhaps the Windows (host) keyboard layout matters?

**Comment by maxpat78** [comment]
No, it does not... setting up US kb layout in Windows 10 didn't change the situation!

**Comment by maron2000** [comment]
I currently have no idea.
Those I previously tried was on a Windows 11 desktop with a JP keyboard,
but I tried again on a Windows 10 Laptop, JP keyboard and still works fine.

**Comment by maxpat78** [comment]
Thanks for your efforts @maron2000 , we'll see if anyone else has run into the same problem!
